ci/kokoro/docker/build-in-docker-cmake.sh

```diff
@@ -424,6 +424,29 @@ if [[ "${TEST_INSTALL:-}" = "yes" ]]; then
     /bin/false
   fi
 
+  io::log_yellow "Verify no extraneous files were installed."
+  # This script is only run on a Fedora-based build, so it's fine that this
+  # setting is not portable.
+  export PKG_CONFIG_PATH="/var/tmp/staging/lib64/pkgconfig:/usr/local/lib64/pkgconfig"
+
+  # Get the version of one of the libraries. These should all be the same, so
+  # it does not matter what we use.
+  GOOGLE_CLOUD_CPP_VERSION=$(pkg-config storage_client --modversion) || true
+  GOOGLE_CLOUD_CPP_VERSION_MAJOR=$(echo "${GOOGLE_CLOUD_CPP_VERSION}" | cut -d'.' -f1)
+
+  # @E is a parameter substitution that provides an escaped version of the
+  # string, i.e 1.21.0 functionally becomes 1\.21\.0.
+  GOOGLE_CLOUD_CPP_VERSION_ESCAPED="${GOOGLE_CLOUD_CPP_VERSION@E}"
```

Comment:
TIL about `@E` -- cool thanks! But can you explain a bit more about why it's needed here? I was playing w/ it in the shell and I couldn't figure out how it did anything interesting with backslashes.

Reply:
Done. When testing, I didn't really see any "changes" either but it's necessary to ensure it's escaped properly in the regex. i.e, `1.21.0` doesn't provide the correct behavior versus `1\.21\.0`.

Reply:
Actually I decided to delete it. I tested it again and see no difference. I think I added it while debugging and thought it was needed but it's not.

Reply:
OK. Maybe I'm holding it wrong, but I'm not seeing that `@E` does anything.

```shell
$ echo $BASH_VERSION
5.1.0(1)-rc1
$ FOO="1.2.3"
$ echo 1x2x3 | grep -E "${FOO@E}"  # If this escaped to 1\.2\.3, then the pattern should not match
1x2x3
$
```

Do you see what I'm doing wrong?

Reply:
My apologies for the misunderstanding. The bug I encountered was fixed by a different section of code and the parameter substitution was not needed. Removed.

Reply:
`@E` doesn't do escaping ... it does expansion of backslash encodings.  For example,
```
$ FOO='\b'
$ echo ${FOO} | od -c
0000000   \   b  \n
0000003
$ echo ${FOO@E} | od -c
0000000  \b  \n
0000002
```

Reply:
Yes I completely misunderstood how the feature worked. Thanks for the clarification.